packages/cli/src/services/communityPackages.service.ts

```diff
@@ -24,6 +24,9 @@ import type { CommunityPackages } from '@/Interfaces';
 import { LoadNodesAndCredentials } from '@/LoadNodesAndCredentials';
 import { Logger } from '@/Logger';
 import { OrchestrationService } from './orchestration.service';
+import { License } from '@/License';
+
+const DEFAULT_REGISTRY = 'https://registry.npmjs.org';
```

Comment:
Let's update the docs that custom registries require a license?

Reply:
[will do](https://github.com/n8n-io/n8n-docs/pull/2328#discussion_r1715449176)